bin/select_nam_nowcast.pl

```diff
@@ -91,8 +91,10 @@
     $numnotfound++;
 }
 # remove later nowcast cycles if they are after the
-# forecast we want to run
-if ( $foundit == 1 ) {
+# forecast we want to run ... unless we are nowcasting
+# from files on the filesystem, in which case we want
+# to keep all the files
+if ( $foundit == 1 && $jshash_ref->{"siteHost"} ne "filesystem" ) {
```

Comment:
```suggestion
if ( $foundit == 1 and $jshash_ref->{"siteHost"} ne "filesystem" ) {
```

While not incorrect here, the lower precedent `and` is what you want in most cases.